integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -246,15 +279,7 @@ private int getattrInternal(String path, FileStat stat) {
       }
       long size = status.getLength();
       if (!status.isCompleted()) {
-        if (mCreateFileEntries.contains(PATH_INDEX, path)) {
-          // Alluxio master will not update file length until file is completed
-          // get file length from the current output stream
-          CreateFileEntry<FileOutStream> ce = mCreateFileEntries.getFirstByField(PATH_INDEX, path);
-          if (ce != null) {
-            FileOutStream os = ce.getOut();
-            size = os.getBytesWritten();
-          }
-        } else if (!AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri)) {
+        if (!AlluxioFuseUtils.waitForFileCompleted(mFileSystem, uri)) {
```

Comment:
This will cost some times for waiting for the file completed, the time of timeout is 5 second, so we hope to use the os.getBytesWritten() as the size, but for this changing, It will always wait for the 5 second for the incomplete file which being right wrote now.